### PR TITLE
Host frontend assets in S3 [fixes #7734]

### PR DESCRIPTION
**Pull request by anarchivist**
Wait to merge until I've confirmed on my testing.

### CODE REVIEW

**Comment by anarchivist** [comment]
This, along with the two related pull requests referenced, is ready for review.

File: config/settings.local.yml.example [hunk 1]
```diff
@@ -50,4 +50,10 @@ widgets:
     twitter: https://twitter.com/exampleorg
     tumblr: http://exampleorg.tumblr.com/
     rss: http://dev.example.org/info/feed
-
+fog:
+  credentials:
+    provider: AWS
+    aws_access_key_id: changeme
+    aws_secret_access_key: changeme
+  directory: changeme
```
Comment (markbreedlove) [inline config/settings.local.yml.example]: Is this the S3 bucket name?

Comment (anarchivist) [inline config/settings.local.yml.example]: Yes. This maps to the Carrierwave `fog_directory` configuration key (see [here](https://github.com/carrierwaveuploader/carrierwave/#using-amazon-s3)).

**Comment by markbreedlove** [comment]
I'm fine with this!  Do you need to do any more testing?

**Comment by anarchivist** [comment]
@markbreedlove I've tested in staging, but feel free to test it yourself if you'd like.

**Comment by AudreyAltman** [comment]
Looks good to me.  :+1: I was not able to test this locally, but the code appears solid.

**Comment by AudreyAltman** [comment]
@anarchivist I am happy with merging if you are satisfied with your testing.

**Comment by anarchivist** [comment]
@AudreyAltman OK, i think this is ready to merge, then.

**Pull request merged**